lib/src/boilerplate_suggestors/boilerplate_utilities.dart

```diff
@@ -278,10 +352,60 @@ String getPropsClassNameFromFactoryDeclaration(
       ?.toSource();
 }
 
+/// Returns the name of the mixin that the [originalClassName] was converted to, or the [originalClassName]
+/// if no matching key is found within [ClassToMixinConverter.convertedClassNames] on the provided [converter] instance.
+String getConvertedClassMixinName(
+    String originalClassName, ClassToMixinConverter converter) {
+  const reservedBaseClassNames = {
+    'FluxUiProps': 'FluxUiPropsMixin',
+    'BuiltReduxUiProps': 'BuiltReduxUiPropsMixin',
+  };
+
+  // If it is a "reserved" base class that the consumer doesn't control / won't be converted as
+  // part of running the codemod in their repo, return the new "known" mixin name.
+  if (reservedBaseClassNames.containsKey(originalClassName)) {
+    return reservedBaseClassNames[originalClassName];
+  }
+
+  return converter.convertedClassNames[originalClassName] ?? originalClassName;
+}
+
 extension IterableAstUtils on Iterable<NamedType> {
   /// Utility to join an `Iterable` based on the `name` of the `name` field
   /// rather than the `toString()` of the object.
-  String joinByName({String separator}) {
-    return map((t) => t.name.name).join('${separator ?? ','} ');
+  String joinByName(
```

Comment:
This method now does much more than what it did originally, and shou,d have its name and doc comment updated.